dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1109,10 +1108,10 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 					return;
 				}
 			} else {
-				accountId = player->GetParentUser()->GetAccountID();
+				if (player->GetCharacter() && player->GetCharacter()->GetParentUser()) accountId = player->GetCharacter()->GetParentUser()->GetAccountID();
```

Comment:
In general you should use the pointer you checked against null, not the next pointer from the function that is side-effect free in theory

Reply:
these getters are inlined to just access the field but alright

Reply:
wasnt any snide remark meant here.  May have come off as that, sorry.